media-tv/gentoo-vdr-scripts/gentoo-vdr-scripts-2.8-r1.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit tmpfiles user-info
+
+DESCRIPTION="Scripts necessary for use of VDR as a set-top-box"
+HOMEPAGE="https://gitweb.gentoo.org/proj/gentoo-vdr-scripts.git/about/"
+SRC_URI="https://gitweb.gentoo.org/proj/gentoo-vdr-scripts.git/snapshot/${P}.tar.bz2 -> ${P}.tbz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~x86"
+
+DEPEND="
+	acct-group/vdr
+	acct-user/vdr"
+RDEPEND="${DEPEND}
+	app-admin/sudo
+	sys-process/wait_on_pid"
+BDEPEND="${DEPEND}"
+
+PATCHES=( "${FILESDIR}/systemd-fix-environment.patch" )
+
+pkg_setup() {
+	VDR_HOME="$(egethome vdr)"
+	echo "VDR_HOME=$VDR_HOME"
+}
+
+src_unpack() {
+	default
+	cp "${FILESDIR}"/systemd-tmpfiles.conf "${WORKDIR}/${PN}.conf" || die
```

Comment:
Maybe this can be moved to `src_prepare` or `src_install` so you don't have to define `src_unpack` in this ebuild at all? Besides using `${T}` sounds more appropriate for this.

Reply:
moved to `src_prepare`